webui-src/index.html

```diff
@@ -1,11 +1,17 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>RetroShare</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
```

Comment:
Just wondering, would it be better to store fonts locally as assets instead of fetching from google everytime? We do that with font-awesome.

I'm saying because the webUI doesn't make any network requests to 3rd parties currently, and this would change that. @csoler any thoughts?

Reply:
I dont' like the idea of the webui doing some requests on the web. Especially if the webui is running on a Tor node, that would kill the anonymity of the node.

Reply:
Surely, I will do it.

My bad, I will keep this in mind next time @csoler @rottencandy

Reply:
> Surely, I will do it.
> 
> My bad, I will keep this in mind next time @csoler @rottencandy

I have added the fonts locally, but we will have to add routes for the font-files in the backend (libretroshare).

So, Let's hold this PR and I will send a PR to libretroshare containing the route for font files. 

After that we can merge this? Or Can we merge now? as It won't affect the webui since browser will fallback to default font. It will show correct font after routes get added.